zerver/tests/test_urls.py

```diff
@@ -103,6 +103,21 @@ def test_get_gcid_when_configured(self) -> None:
             self.assertEqual('success', data['result'])
             self.assertEqual('ABCD', data['google_client_id'])
 
+    def test_config_error_endpoints_dev_env(self) -> None:
+        '''
+        The content in these pages has already been tested
+        by code in the test_auth_backends.py. So, here we
+        just check if all URLs load for URL test coverage.
+        '''
+        auth_types = ['apple', 'google', 'github', 'gitlab', 'ldap', 'dev', 'saml',
```

Comment:
I think we might need to figure out a way to automatically generate this list. Needing to manually keep this list in this test updated will not be fun.

Reply:
used the `Realm.AUTHENTICATION_FLAGS` here